activerecord/test/cases/associations/has_one_associations_test.rb

```diff
@@ -540,16 +540,18 @@ def test_create_association_replaces_existing_without_dependent_option
     assert_not_equal ships(:black_pearl), new_ship
     assert_equal new_ship, pirate.ship
     assert_predicate new_ship, :new_record?
+    assert_predicate new_ship, :invalid?
```

Comment:
Maybe assert for the error here for clarity? Similar to line 655.

Reply:
I've addressed.

Reply:
Hmm, looking at it again with fresh eyes I think just the following is good enough:
```
assert_predicate new_ship, :invalid?
```
Sorry 😞 
Could you remove the assertion on the errors again?
After that I think this is good to go.

Reply:
Removed [here](https://github.com/rails/rails/compare/045907cc63f0f4bdf59367349c6adb86921fef8f..e61234dac78f5a77712b853b2ccf5c3963a54a06).